tests/testthat/test-dirs.R

```diff
@@ -25,11 +25,11 @@ test_that("directory functions returns as expected", {
   expect_s3_class(x, "tbl_df")
   expect_equal(dim(x), c(6, 3))
 
-  msg <- paste0(
+  suppressMessages(msg <- message(
```

Comment:
You don't need to do that. You just need to escape the parentheses in your expected string with \( and \).